test/index.js

```diff
@@ -3823,6 +3823,23 @@ describe('list', function() {
 
   });
 
+  describe('append', function() {
+
+      it('adds the element to the end of the list', function() {
+         eq(S.append('c', ['a', 'b']), ['a', 'b', 'c']);
+         eq(S.append({x:3}, [{x:1}, {x:2}]), [{x:1}, {x:2}, {x:3}]);
+      });
+
+      it('adds a list to a list of lists', function() {
+          eq(S.append([3, 4], [[1], [2]]), [[1], [2], [3, 4]]);
+      });
+
+      it('works on empty list', function() {
+        eq(S.append(1, []), [1]);
+      });
+
+  });
+
```

Comment:
The indentation here is inconsistent. The body of the `it` blocks are indented by 3, 4, and 2 spaces respectively. The body of the `describe` block is indented by 4 spaces. Let's use 2 spaces in all cases.

This makes me think checking indentation in #183 is actually a good idea. We could enable it for __test/index.js__ but not for __index.js__.